src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -366,7 +366,7 @@ def list(self, daemon_name=None):
 
     def get(self, uid, daemon_name=None):
         # type: (str, Optional[str]) -> dict
-        result = self.proxy(daemon_name, 'GET', 'user', {'uid': uid})
+        result = self.proxy(daemon_name, 'GET', 'user', {'uid': uid, 'stats': True})
```

Comment:
Here we're enforcing retrieval of stats (required for Dashboard) but in the future we should consider to make it optional so third-party API consumers can choose if they want stats or not.